administrator/language/en-GB/com_config.ini

```
@@ -70,11 +70,17 @@ COM_CONFIG_FIELD_ERROR_REPORTING_LABEL="Error Reporting"
 COM_CONFIG_FIELD_FEED_EMAIL_LABEL="Feed Email Address"
 COM_CONFIG_FIELD_FILESYSTEM_PATH_DESC="The filesystem path where session data will be stored.<br>If empty, the system's temporary directory will be used."
 COM_CONFIG_FIELD_FILESYSTEM_PATH_LABEL="Session Save Path"
-COM_CONFIG_FIELD_FILTERS_CUSTOM_BLACK_LIST="Custom Blacklist"
-COM_CONFIG_FIELD_FILTERS_DEFAULT_BLACK_LIST="Default Blacklist"
+COM_CONFIG_FIELD_FILTERS_CUSTOM_DISALLOW_LIST="Custom DisallowList"
+COM_CONFIG_FIELD_FILTERS_DEFAULT_DISALLOW_LIST="Default DisallowList"
+; Deprecate in Joomla! 4, remove in Joomla! 5
+COM_CONFIG_FIELD_FILTERS_CUSTOM_BLACK_LIST="Custom DisallowList"
```

```suggestion
COM_CONFIG_FIELD_FILTERS_CUSTOM_BLACK_LIST="Custom Disallow List"
```
Formalise the wording to show as separate words.